app/code/Magento/LayeredNavigation/Test/Mftf/Test/StorefrontLayeredNavigationOnHomePageTest.xml

```diff
@@ -0,0 +1,80 @@
+            <createData entity="productAttributeOption" stepKey="createProductAttributeOption1">
+                <requiredEntity createDataKey="createProductAttribute"/>
+            </createData>
+            <createData entity="productAttributeOption" stepKey="createProductAttributeOption2">
+                <requiredEntity createDataKey="createProductAttribute"/>
+            </createData>
+            <getData entity="ProductAttributeOptionGetter" index="1" stepKey="getProductAttributeOption1">
+                <requiredEntity createDataKey="createProductAttribute"/>
+            </getData>
+            <getData entity="ProductAttributeOptionGetter" index="2" stepKey="getProductAttributeOption2">
+                <requiredEntity createDataKey="createProductAttribute"/>
+            </getData>
+            <createData entity="AddToDefaultSet" stepKey="createProductAddToAttributeSet">
+                <requiredEntity createDataKey="createProductAttribute"/>
+            </createData>
+            <createData entity="ApiSimpleProductWithCategory" stepKey="createSimpleProduct1">
+                <requiredEntity createDataKey="createProductAttribute"/>
+                <requiredEntity createDataKey="getProductAttributeOption1"/>
+                <requiredEntity createDataKey="createCategory"/>
+            </createData>
+            <createData entity="ApiSimpleProductWithCategory" stepKey="createSimpleProduct2">
+                <requiredEntity createDataKey="createProductAttribute"/>
+                <requiredEntity createDataKey="getProductAttributeOption2"/>
+                <requiredEntity createDataKey="createCategory"/>
+            </createData>
+            <magentoCLI command="config:set web/default/front catalog/category/view/id/$createCategory.id$" stepKey="setHomepageConfig"/>
+
+            <magentoCron groups="index" stepKey="reindexInvalidatedIndices"/>
+            <actionGroup ref="CliCacheCleanActionGroup" stepKey="cleanInvalidatedCaches">
+                <argument name="tags" value="config full_page"/>
+            </actionGroup>
+        </before>
+
+        <after>
+            <magentoCLI command="config:set web/default/front cms" stepKey="unsetHomepageConfig"/>
+            <deleteData createDataKey="createSimpleProduct2" stepKey="deleteSimpleProduct2"/>
+            <deleteData createDataKey="createSimpleProduct1" stepKey="deleteSimpleProduct1"/>
+            <deleteData createDataKey="createProductAttribute" stepKey="deleteAttribute"/>
+            <deleteData createDataKey="createCategory" stepKey="deleteCategory"/>
+
+            <magentoCron groups="index" stepKey="reindexInvalidatedIndices"/>
+            <actionGroup ref="CliCacheCleanActionGroup" stepKey="cleanInvalidatedCaches">
+                <argument name="tags" value="config full_page"/>
+            </actionGroup>
+        </after>
+
+        <amOnPage url="{{StorefrontHomePage.url}}" stepKey="goToStorefrontHomePage"/>
```

Comment:
Hi @Bartlomiejsz . 
You can use `StorefrontOpenHomePageActionGroup` instead .

Reply:
Done @Usik2203